app/subscriber/src/features/my-reports/edit/settings/template/utils/generateChartOptions.ts

```diff
@@ -0,0 +1,116 @@
+import { IChartSectionSettingsModel } from 'tno-core';
+
+import { IChartData } from './IChartData';
+
+export const generateChartOptions = (
```

Comment:
There is a similar function in tno-core called `mergeChartSettings` that I would like to combine so that we don't have similar logic in two places.